content/streamlit-cloud/get-started/deploy-an-app/index.md

```diff
@@ -17,20 +17,40 @@ If you want to deploy your app on a different cloud service, check out the [Depl
 
 Streamlit Community Cloud launches apps directly from your GitHub repo, so your app code and dependencies need to be on GitHub before you try to deploy the app. See [App dependencies](/streamlit-community-cloud/get-started/deploy-an-app/app-dependencies) for more information.
```

Comment:
Maybe turn the last sentence into "For more information on how to specify dependencies, see [App dependencies](...)"